src/pages/BanquetFacility/BanquetFacility.js

```diff
@@ -3,6 +3,11 @@ import Layout from 'components/Layout/Layout.js';
 import Container from 'components/Container/Container.js';
 import Button from 'components/Button/Button.js';
 import Image from 'components/Image/Image.js';
+import bfimg1 from 'assets/banquet-facility1.jpg';
+import bfimg2 from 'assets/banquet-facility2.jpg';
+import bfimg3 from 'assets/banquet-facility3.jpg';
+import bfimg4 from 'assets/banquet-facility4.jpg';
```

Comment:
We wont be handling images this way - this is importing the image into the JS which will cause the JS to slow and be blocking - we should host these images somewhere and use the link to them for better performance. Doing it this way is tell JS to try and use the image binary data (which is not good)